data/tools/converters.yml

```diff
@@ -384,3 +388,279 @@
+        comment: Extract ISA-Tab compatible metadata from BIDS
+        license: Apache-2.0
+
+    -   name: BIDSto3col
+        url: https://github.com/bids-standard/bidsutils/tree/master/BIDSto3col
+        data_types:
+        -   TSV
+        expected_input:
+        -   BIDS
+        language:
+        -   shell
+        -   Python
+        comment: Reads BidsTSV and then creates 3 column event files, one per event type if a "trial_type" column is found.
+
+    -   name: BIDS2NDA
+        url: https://github.com/bids-standard/BIDS2NDA
+        data_types:
+        -   BIDS
+        expected_input:
+        -   BIDS
+        language:
+        -   Python
+        comment: Extract NIHM Data Archive compatible metadata from BIDS
+        license: Apache-2.0
+
+    -   name: bids2xar - for XNAT import
+        url: https://github.com/lwallace23/bids2xar
+        data_types:
+        -   MRI # not sure
+        expected_input:
+        -   BIDS
+        language:
+        -   Python
+        comment: Convert BIDS data set into XNAT XAR bundles
+
+    -   name: BIDS2NIDM
+        url: https://github.com/incf-nidash/PyNIDM/blob/master/nidm/experiment/tools/nidm2bids.py
+        data_types:
+        -   BIDS
+        expected_input:
+        -   BIDS
+        language:
+        -   Python
+        comment: |
+            This program will convert a NIDM-Experiment RDF document to a BIDS
+            dataset.
+
+    -   name: AFNI BIDS-tools
+        url: https://github.com/nih-fmrif/BIDS-tools
+        data_types:
+        -   MRI
+        expected_input:
+        -   BIDS # not sure about that one
+        language:
+        -   shell
+        -   Python
+        comment: ARCHIVED - Scripts, tools, and documents on creating, parsing, and working with BIDS-structured data sets.
+
+
+# Miscellaneous
+# Comment: Not exactly BIDS converters but are common tools that can used by other BIDS converters.
+    -   name: convert-eprime
+        url: https://github.com/tsalo/convert-eprime
+        data_types:
+        -   MISC
+        -   behavioral
+        expected_input:
+        -   E-Prime log files
+        language:
+        -   Python
+        comment: Python functions to convert E-Prime files to CSVs. Not currently being developed.
+        license: MIT
+
+    -   name: DCM2NIIx
+        url: https://github.com/neurolabusc/dcm2niix
+        data_types:
+        -   MISC
+        -   DICOM
+        -   PAR/REC
+        expected_input:
+        -   ???
+        language:
+        -   C++ (compiled versions available from <ahref="https://www.nitrc.org/plugins/mwiki/index.php/dcm2nii:MainPage">NITRC</a>)
+        comment: dcm2nii DICOM to NIfTI converter
+
+    -   name: DICM2NII
+        url: https://github.com/xiangruili/dicm2nii
+        data_types:
+        -   MISC
+        -   DICOM
+        expected_input:
+        -   DICOM
+        language:
+        -   MATLAB
+        comment: dcm2nii DICOM to NIfTI converter
+        license: BSD-2
```

Comment:
- I opened an issue to remove those: https://github.com/bids-standard/bids-website/issues/519

They do not fall under proper BIDS converter for me, they are dicom to nifti converters.

We have too many converters as it is and this adds noise and confuse the users.

So nuke those out.